.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,12 @@
+on: [pull_request]
+name: benchmark pull requests
+jobs:
+  runBenchmark:
+    name: run benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: run benchmark
+        uses: matchai/criterion-compare-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You don't need this. I think.

Reply:
Er, this is the part that makes use of criterion-compare-action, I'm fairly sure it won't work otherwise

Reply:
I looked at the code and it looks like it is not being used anywhere except creating comment. Creating comment should be done by anyone and don't need GitHub token. 


Let's merge this first and then experiment

Reply:
Without it, we would have only the checkout left and I doubt it would run anything at all

Reply:
Without it, we would have only the checkout left and I doubt it would run anything at all

Reply:
This action is really the one making the `cargo bench` run, see https://github.com/matchai/criterion-compare-action/blob/0270c5b81e68b101cd0f7c959ffe9dd320a69bc1/entrypoint.js#L6-L13 so yeah, it's mandatory